src/bin/customer/close.rs

```diff
@@ -149,7 +151,7 @@ async fn finalize_customer_close(
     channel_name: &ChannelName,
     merchant_balance: MerchantBalance,
 ) -> anyhow::Result<()> {
-    // TODO: assert that the db status is PendingClose,
+    // TODO: assert that the db status is PendingClose?
```

Comment:
Is this still a TODO? Or did this just become a question? It seems somewhat related to this issue? Can we either resolve the TODO or create an issue (or just link an issue) to make sure this is tracked?

Reply:
This is linked to #167. I am no longer sure if that issue is something we still should do, but we can have that discussion over there.